build.sh

```diff
@@ -3,37 +3,44 @@
 set -eux
 
 # Location of PWD and package source directory.
-pkg_root=`dirname $(readlink -f $0)`
+pkg_root=$(cd "$(dirname "$(readlink -f -n "${BASH_SOURCE[0]}" )" )" && pwd -P)
 
 INSTALL_TARGET=${INSTALL_TARGET:-"wcoss2"}
-INSTALL_PREFIX=${INSTALL_PREFIX:-"$pkg_root/install"}
+INSTALL_PREFIX=${INSTALL_PREFIX:-"${pkg_root}/install"}
 MODULEFILE_INSTALL_PREFIX=${MODULEFILE_INSTALL_PREFIX:-"${INSTALL_PREFIX}/modulefiles"}
 
-target=$(echo $INSTALL_TARGET | tr [:upper:] [:lower:])
-if [[ "$target" =~ ^(wcoss2|hera|orion|jet)$ ]]; then
-  source $pkg_root/versions/build.ver
+target="${INSTALL_TARGET,,}"
+if [[ "${target}" =~ ^(wcoss2|hera|orion|jet|hercules)$ ]]; then
+  # copy the target specific build.ver and run.ver
+  cp "${pkg_root}/versions/build.${target}.ver" "${pkg_root}/versions/build.ver"
+  cp "${pkg_root}/versions/run.${target}.ver" "${pkg_root}/versions/run.ver"
```

Comment:
Links would probably be better here.

Reply:
Doesn't ops require copies and not symlinks?

Reply:
copying instead of linking.  can update later if necessary. Ok?

Reply:
Sure